cypress/elements/shared/sharedElements.js

```diff
@@ -24,6 +24,9 @@ export const SHARED_ELEMENTS = {
   },
   warningDialog: '[data-test-id="warning-dialog"]',
   pageHeader: "[data-test-id='page-header']",
+  multiAutocomplete: {
+    selectedOptions: '[id*="selected-option-"]',
```

Comment:
Why isn't it `data-test-id` instead?

Reply:
@orzechdev `data-test-id` is one of the option and if an element has it's own `id` then we don't have to add separate attribute in this element.